NEWS.md

```diff
@@ -0,0 +1,1908 @@
+# wolfSSL Release 3.15.0 (05/01/2018)
```

Comment:
This is a new file with stuff in it, why is not in a .ac file?

Reply:
Oops. Fixed that. Checking for other missed files, I found a bunch of sources and certificates with the execute bit set. I also updated the rpm build.